<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataHub_image</class>
 <widget class="QWidget" name="dataHub_image">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="main_layout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Brightfield Image</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTreeWidget" name="tree">
            <property name="selectionMode">
             <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
            </property>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="btn_save">
              <property name="text">
               <string>Save all (.db)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_load">
              <property name="text">
               <string>Load (.db)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_remove">
              <property name="text">
               <string>Remove selected</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_save_png">
              <property name="text">
               <string>Save selected units (.png)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
